corehq/apps/callcenter/sync_usercase.py

```diff
@@ -226,32 +227,35 @@ def _call_center_location_owner(user, ancestor_level):
     return owner_id
 
 
-def sync_usercase(user):
+def sync_usercase(user, domain):
     with CriticalSection(get_sync_lock_key(user._id)):
-        _UserCaseHelper.commit(list(_iter_sync_usercase_helpers(user)))
+        domain_obj = Domain.get_by_name(domain)
+        _UserCaseHelper.commit(list(_iter_sync_usercase_helpers(user, domain_obj)))
 
 
-def _iter_sync_usercase_helpers(user):
-    if user.project.usercase_enabled:
+def _iter_sync_usercase_helpers(user, domain_obj):
+    if domain_obj.usercase_enabled:
         yield _get_sync_usercase_helper(
             user,
+            domain_obj.name,
             USERCASE_TYPE,
             user.get_id
         )
 
 
-def sync_usercases(user):
+def sync_usercases(user, domain):
     """
     Each time a CommCareUser is saved this method gets called and creates or updates
     a case associated with the user with the user's details.
 
     This is also called to create usercases when the usercase is used for the
     first time.
     """
+    domain_obj = Domain.get_by_name(domain)
     with CriticalSection(get_sync_lock_key(user._id)):
         helpers = list(chain(
-            _iter_sync_usercase_helpers(user),
-            _iter_call_center_case_helpers(user),
+            _iter_sync_usercase_helpers(user, domain_obj),
+            _iter_call_center_case_helpers(user, domain_obj),
```

Comment:
```suggestion
            _iter_call_center_case_helpers(user, domain_obj) if isinstance(user, CommCareUser) else [],
```

(could also push this down to `_iter_call_center_case_helpers` which might be cleaner)

Reply:
This is because we don't want to support call center features for Web Users correct?

Reply:
yes, that's correct.

Reply:
Doesn't this mean `sync_call_center_user_case` could go back to only accepting mobile users and wouldn't need the domain passed in?